common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerActivity.java

```diff
@@ -62,7 +62,10 @@ protected void onCreate(final Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
         if (savedInstanceState == null) {
-            mBrokerInteractiveRequestIntent = getIntent().getExtras().getParcelable(BROKER_INTENT);
+            Bundle extras = getIntent().getExtras();
+            if(extras != null) {
+                mBrokerInteractiveRequestIntent = extras.getParcelable(BROKER_INTENT);
```

Comment:
Do we understand in what scenarios the BROKER_INTENT is not being passed?
Should we return an error or ignore silently?

Reply:
There is not much information available except for the crash stack. The partner also has closed the investigation due to low hits.
Even though it is good to understand the reason, we should prevent crashes by adding this check for sure.
I do not think we can return error here, but logging may be helpful.

Reply:
If we don't return error here then what happens? What is the end user experience in this case?